docs/source/customizations/overview.mdx

```diff
@@ -10,7 +10,7 @@ You can create **customizations** for the Apollo Router to add functionality tha
 The Apollo Router supports the following customization types:
 
 - [**Rhai scripts**](./rhai/)
-    - The [Rhai scripting language](https://rhai.rs/book/) enables you to add functionality directly to your stock router binary by hooking into different phases of the router's request lifecycle.
+    - The [Rhai scripting language](https://rhai.rs/book/) lets you add functionality directly to your stock router binary by hooking into different phases of the router's request lifecycle.
 - [**External co-processing**](./coprocessor/) ([Enterprise feature](../enterprise-features/))
     - If your organization has a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/), you can write custom request-handling code in any language. This code can run in the same container as your router or separately.
     - The router calls your custom code via HTTP, passing it the details of each incoming client request.
```

Comment:
Why do we recommend using Rhai scripts if possible?

Reply:
it's the easiest customization to deploy: users write a rhai file and load it into the router. Coprocessors need an external service that exposes the API expected by the router, and native plugins requires rebuilding the router